src/overlay.py

```diff
@@ -33,6 +33,7 @@ def emit(self, record):
 
 class Overlay:
     def __init__(self):
+        LOGGER.setLevel(str(IniConfigLoader().advanced_options.log_lvl).upper())
```

Comment:
Why do this? log level is already set in main and all other logger inherit the level

Reply:
For the overlay terminal specifically, the log level is being set to debug at all times. If you check it before this change you'll see it's 0.

Perhaps this is a band-aid fix for something deeper? I don't know, but I know this resolves it.

Reply:
Ok I see. Please change line 104 which sets the level of this handler to

```LOGGER.root.handlers[0].level```

of course guard it if no handlers are available and then just set it to 0

Reply:
I made this change and it appears to work great now, I tested at debug/info/warning levels and all were working as expected.